internal/gengapic/genrest.go

```diff
@@ -603,8 +603,22 @@ func (g *generator) pagingRESTCall(servName string, m *descriptor.MethodDescript
 	p("")
 	p("  unm.Unmarshal(buf, resp)")
 	p("  it.Response = resp")
-	// TODO(dovs): handle map pages
-	p("  return resp.Get%s(), resp.GetNextPageToken(), nil", snakeToCamel(elemField.GetName()))
+	repeatedField, elts := fmt.Sprintf("resp.Get%s()", snakeToCamel(elemField.GetName())), ""
+	if pt.mapValueTypeName != "" {
+		elts = "elts"
```

Comment:
nit: I know Go style advocates for short variable names, so this may be a losing fight, but I'd prefer just `elements` or even `elems`

Reply:
+1 `elems` would be better :)

Reply:
s/elts/elems it is then.